services/src/main/java/org/exoplatform/webconferencing/jitsi/JitsiProvider.java

```diff
@@ -45,6 +55,15 @@
   /** The Constant TYPE. */
   public static final String TYPE                        = "jitsi";
 
+  /** The constant JITSI_CONFIGURATION. */
+  public static final String    JITSI_CONFIGURATION         = "jitsi-configuration";
+
+  /** The constant JITSI_SCOPE_NAME. */
+  protected static final String JITSI_SCOPE_NAME            = "webconferencing.jitsi".intern();
```

Comment:
Don't use `intern` it will reside in the perm gen memory what may lead to not expected effects.

Reply:
Have already removed